src/tests/end-to-end/simple_study/simple-study.cpp

```diff
@@ -24,7 +24,7 @@ struct StudyFixture : public StudyBuilder
 	double loadInArea = 0.;
 	double clusterCost = 0.;
 	ThermalClusterConfig clusterConfig;
-	TimeSeriesConfigurer<Matrix<double, int32_t>> loadTSconfig;
+	TimeSeriesConfigurer<Matrix<double>> loadTSconfig;
```

Comment:
Considering all matrices are `Matrix<double>`, I think we could remove this template argument, and so remove this template as well.